opacus/privacy_engine.py

```diff
@@ -36,6 +37,13 @@
 from torch.utils.data import DataLoader
 
 
+def _is_ew_compatibility_required(grad_sample_mode: str):
```

Comment:
```suggestion
def _is_ew_compatible(grad_sample_mode: str):
```

Reply:
Nope
We need `ew_compatibility_mode`, because initial version of ExpandedWeights didn't handle loss reduction properly, so we needed to amend the gradients at the optimizer level.
With the recent version, optimizer is still EW compatible, but doesn't need to know about EW at all (i.e. no `ew_compatibility_mode` is required)

Reply:
I see. In that case, two minor suggestions:
1. Add the above explanation as a docstring to the function
2. may be make name more explicit? `_is_ew_compatibility_check_required`?

Reply:
yeah, I see how much confusion it creates
I'll follow your suggestions above, but also think we'd better remove `ew_compatibility_mode` completely once torch==1.13 released, dropping support for pre-1.13 `ExpandedWeights`